source/Engine.cpp

```diff
@@ -555,8 +555,8 @@ void Engine::Step(bool isActive)
 
 	const System *currentSystem = player.GetSystem();
 	// Update this here, for thread safety.
-	if(!player.HasTravelPlan() && flagship && flagship->GetTargetSystem())
-		player.TravelPlan().push_back(flagship->GetTargetSystem());
+	// if(!player.HasTravelPlan() && flagship && flagship->GetTargetSystem())
+	//	 player.TravelPlan().push_back(flagship->GetTargetSystem());
```

Comment:
Delete it or leave it or add a comment why it is commented out, but do not leave it just like this

Reply:
Yep, that makes sense. I'll delete it, now that I know it works.

Reply:
I am still not sure if this is safe, thread safety is a very important thing

Reply:
oh, wait, no it should be fine, what he means is do it here instead of somewhere else, sorry I am stupid

Reply:
Oh so this is fine? Got me confused as well so idk